packages/core/src/tech/HlsJsTech.ts

```diff
@@ -97,6 +108,14 @@ export default class HlsJsTech extends BaseTech {
     }
   }
 
+  protected onBitrateChange() {
+    this.emit(PlayerEvent.BITRATE_CHANGE, this.currentLevel);
+  }
+
+  protected onErrorEvent(event, data) {
+    this.emit(PlayerEvent.ERROR, data);
+  }
```

Comment:
We should probably do a utils method to convert hls.js errors and shaka errors to an unified format for our analytics usage.

Reply:
Did for hls.js errors, but the data format for Shaka resulted in little more than variable assignment to errorData